wasm/Cargo.toml

```diff
@@ -10,7 +10,7 @@ crate-type = ["cdylib"]
 [dependencies]
 getrandom = { version = "0.2", features = ["js"] }
 renegade-crypto = { git = "https://github.com/renegade-fi/renegade.git", default-features = false }
-sha2 = "0.10.8"
+sha2 = { version = "0.10.8", features = ["asm"] }
```

Comment:
just making sure the feature flag is supposed to be `"asm"` and not `"wasm"`

Reply:
yup, `"asm"` for "assembly"